dispatch/BESLog.h

```diff
@@ -108,12 +115,10 @@ class BESLog: public BESObj {
 private:
     static BESLog * d_instance;
 
-    int d_flushed;
     std::ofstream * d_file_buffer;
```

Comment:
Initialize this here to nullptr (std::ofstream *d_file_buffer = nullptr;) and remove from the ctor if needed.